docs/json/radarr/cf/anime-bd-tier-06-fansubs.json

```diff
@@ -57,7 +57,7 @@
       "negate": false,
       "required": false,
       "fields": {
-        "value": "\\b(ANE)\\b"
+        "value": "\\[ANE\\]"
```

Comment:
would it make sense to use `-ANE$` as well?

Reply:
Probably yes